postgres/defaults.yaml

```diff
@@ -71,6 +71,8 @@ postgres:
 
   bake_image: False
 
+  manage_force_reload_modules: False
```

Comment:
No need to have this configurable. We will make the default behavior correct, so there would be no undesired state changes.